commonmark-spannable/src/main/java/org/commonmark/renderer/spannable/CoreSpannableProvider.java

```diff
@@ -37,7 +39,8 @@ public Set<Class<?>> getSpanTypes() {
                 LinkSpan.class,
                 OrderedListItemSpan.class,
                 QuoteSpan.class,
-                UnorderedListItemSpan.class)
+                UnorderedListItemSpan.class,
+                StrikethroughSpan.class)
```

Comment:
This is very nit, but I think we were keeping it in alphabetical order 👀

Reply:
👌 Thank you for pointing it out! Will update